guides/common/modules/con_making-open-source-more-inclusive.adoc

```diff
@@ -0,0 +1,4 @@
+[id="making-open-source-more-inclusive_{context}"]
+= Making open source more inclusive
+
+Red Hat is committed to replacing problematic language in our code, documentation, and web properties. Because of the enormity of this endeavor, these changes are being updated gradually and where possible. For more details, see link:https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language[our CTO Chris Wright's message].
```

Comment:
```suggestion
Red Hat is committed to replacing problematic language in our code, documentation, and web properties.
Because of the enormity of this endeavor, these changes are being updated gradually and where possible.
For more details, see https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language[our CTO Chris Wright's message].
```

Reply:
We could also replace "Red Hat" with team, make the third line Satellite-only, and include this statement to all build targets.

Reply:
I'd be against including it in the Foreman project. We had a discussion in https://community.theforeman.org/t/rfc-replacing-blacklist-with-blocklist-in-our-templates/19276 and https://community.theforeman.org/t/terminology-updates-in-foreman/19477 in the past. Tomer made a few small changes, but the biggest effect is that a very valued contributor left the project. IMHO it's only virtue signaling. Adding this would only distract from the real documentation.

Reply:
I (think I) know who left and thought about them when reading this too. But still, I strongly support making an effort to use inclusive language in the docs. I help maintain the docs and review/support this PR to reword ~blacklist~ to _blocklist_ etc.

I am OK either way, but it basically leads to the question _who_ can speak for the Foreman Community in terms of "who decides if we can write 'Foreman supports inclusive language'". It's not like every other (problematic) change is discussed before implementing it.

I am OK with creating a small RFC or keeping it as is.

Reply:
I think there's a difference in including the header and making changes. I'm firmly against including the header in upstream. The other individual changes can be judged separate.

Reply:
OK, then let's keep it as suggestion by @apinnick

Reply:
@maximiliankolb 
I do not recommend adding an inclusive language module while your default branch is still "master". :-)
That is why I marked this module for Satellite only.

Re: Not including it in the Foreman repo at all. I could write a script to inject this module into downstream adoc files. I added it up stream when I saw that the "Feedback" module is included.

Will fix the lines.

Reply:
I meant to include it in the rendered upstream docs, not repo

Reply:
@ekohl Has anyone ever proposed renaming the default branches in Foreman and Foreman docs? Most of our  projects, like [openshift-docs](https://github.com/openshift/openshift-docs), use `main` instead of `master`.

Reply:
@apinnick foreman.git has used `develop` for the past 10+ years but I don't think anyone wants to put in the effort to rename other branches. It's utterly painful and I don't see a good reason to.

Reply:
@ekohl  
> I meant to include it in the rendered upstream docs, not  repo

I thought this comment meant that you were considering including the new module in rendered upstream docs. 
If that is the case, I was pointing out that it would look odd because the upstream default branch is 'master'. I agree that it is not a trivial matter to rename branches.